modules/command-engine/javadsl/src/main/scala/surge/javadsl/common/AggregateRefBaseTrait.scala

```diff
@@ -24,8 +24,8 @@ trait AggregateRefBaseTrait[AggId, Agg, Cmd, Event] extends AggregateRefTrait[Ag
     FutureConverters.toJava(queryState.map(_.asJava))
   }
 
-  def applyEvent(event: Event): CompletionStage[ApplyEventResult[Agg]] = {
-    val envelope = PersistentActor.ApplyEvent[Event](aggregateId.toString, event)
+  def applyEvents(events: Seq[Event]): CompletionStage[ApplyEventResult[Agg]] = {
```

Comment:
I think we want this to remain backwards compatible from here with both the old `applyEvent` and new `applyEvents` method

Reply:
Aah never mind, I see now :)